### DISCUSSION TITLE
Industry Session Status

### DISCUSSION THREAD

**Discussion by darshan-desale** [github/india#202] [GitHub Platform]
I have watched the first session among Industry session, Although Completion Status is shown as Incomplete. Please Tell me, What should I do?

**Comment by TheSuranaverse** [comment]
@darshan9396 Is your doubt solved?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by FRTMentor** [comment]
Please watch the complete video without skipping any sections and without forward the video.
